wouso/games/teamquest/models.py

```diff
@@ -14,7 +15,11 @@ def is_head(self):
 
 
 class TeamQuest(models.Model):
-	pass
+    start = models.DateTimeField()
+    end = models.DateTimeField()
+    title = models.CharField(default="", max_length=100)
+    questions = models.ManyToManyField(Question)
```

Comment:
The questions are now part of the level (TeamQuestLevel). We don't need this anymore.